package/python3-img-proof.spec

```diff
@@ -60,7 +60,7 @@ Requires:       python3-azure-mgmt-network
 Requires:       python3-azure-mgmt-resource
 Requires:       python3-boto3
 Requires:       python3-click
-Requires:       python3-google-api-python-client
+Requires:       python3-google-cloud-compute
```

Comment:
We are going to want to change the packaging to the same scheme as `ec2imgutils`

Reply:
Ideally this builds for TW which wouldn't necessarily be python3.11 🤔

Reply:
The `Requires:` directives should just be `python-*` now. With the assumption that the dependencies use the macros correctly. Then when this code gets build for python311 for example it will automatically pull the python311 dependencies.

Reply:
Updated, was looking at an old version of ec2imgutils.

Reply:
> The `Requires:` directives should just be `python-*` now.

rly?